sdk-core/src/main/java/com/reach5/identity/sdk/core/models/RequestModels.kt

```diff
@@ -53,11 +61,12 @@ data class RequestPasswordResetRequest (
     val redirectUrl: String? = null,
     @SerializedName("phone_number")
     val phoneNumber: String?
-)
+) : Parcelable
 
+@Parcelize
```

Comment:
Were these data classes parcelized before?

Reply:
Nope there were not.

Reply:
Why are we parcelizing them if they weren't ?